src/test/java/seedu/address/logic/commands/FindTagCommandTest.java

```diff
@@ -52,25 +66,57 @@ void equals() {
     }
 
     @Test
-    public void execute_zeroKeywords_noPersonFound() {
+    public void execute_singleKeywordNoActivatedTags_noPersonFound() {
         String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 0);
-        TagContainsKeywordsPredicate predicate = preparePredicate(" ");
-        FindTagCommand command = new FindTagCommand(predicate);
+        model.addTag(VALID_TAG_COLLEAGUES);
+        List<String> singleKeyword = new ArrayList<>();
+        singleKeyword.add(VALID_TAGNAME_COLLEAGUES);
+        FindTagCommand command = new FindTagCommand(singleKeyword);
+        TagContainsKeywordsPredicate predicate = new TagContainsKeywordsPredicate(singleKeyword);
+        expectedModel.addTag(VALID_TAG_COLLEAGUES);
         expectedModel.updateFilteredPersonList(predicate);
         assertCommandSuccess(command, model, expectedMessage, expectedModel);
         assertEquals(Collections.emptyList(), model.getFilteredPersonList());
     }
 
     @Test
-    public void execute_singleKeyword_multiplePersonsFound() {
+    public void execute_zeroKeywordsNoActivatedTags_throwsCommandException() {
+        String expectedMessage = FindTagCommand.MESSAGE_USAGE;
+        List<String> emptyKeywords = new ArrayList<>();
+        FindTagCommand command = new FindTagCommand(emptyKeywords);
+        TagContainsKeywordsPredicate predicate = new TagContainsKeywordsPredicate(emptyKeywords);
+        expectedModel.updateFilteredPersonList(predicate);
```

Comment:
I think you don't need to have this line here!

Reply:
Yup you're right! Removed them!